code/style.css

```diff
@@ -1,4 +1,372 @@
+/*general*/
```

Comment:
Try to structure your CSS a little bit as well. Maybe it would make sense to style the `body` right after the `html` because that's when it appears in Document Object Model